unity-renderer/Assets/Scripts/MainScripts/DCL/Components/AvatarAttach/AnchorPointsGetterHandlers/GetAnchorPointsHandler.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace DCL.Components
 {
-    internal class GetAnchorPointsHandler : IDisposable
+    public class GetAnchorPointsHandler : IDisposable
```

Comment:
could we move this file to the ecs7 component folder since the previous component will be deprecated?